src/components/dialog/content/manager/ManagerDialogContent.vue

```diff
@@ -162,15 +162,16 @@ watchEffect(async () => {
   }
 })
 
-const resultsWithKeys = computed(() =>
-  displayPacks.value.map((item) => ({
-    ...item,
-    key: item.id || item.name
-  }))
+const resultsWithKeys = computed(
+  () =>
+    displayPacks.value.map((item) => ({
+      ...item,
+      key: item.id || item.name
+    })) as (components['schemas']['Node'] & { key: string })[]
 )
 
```

Comment:
For this it's just complaining that the key can be undefined. Since we only need the key for the purpose of rendering, maybe we can just generate a key if the name doesn't exist

```suggestion
const resultsWithKeys = computed(() =>
  displayPacks.value.map((item) => ({
    ...item,
    key: item.id || item.name || useId()
  }))
)
```

Will also need to import `useId` or `generateUUID`